lib/fluent/plugin/filter_process_redfishalert.rb

```diff
@@ -11,9 +11,12 @@ class ProcessRedfishAlert < Filter
     config_param :coloregion, :string
     config_param :username, :string
     config_param :passwordFile, :string
+    config_param :hardware, :string, :default => "SDFLEX"
 
     def configure(conf)
       super
+        @hwtDeviceURI = Hash["Dell_PowerEdge_iDRAC"=>"Systems/System.Embedded.1", "SDFLEX" => "Chassis/RMC"]
+        @deviceRackURI = Hash["Dell_PowerEdge_iDRAC"=>"Systems/System.Embedded.1", "SDFLEX" => "Chassis/RackGroup"]
```

Comment:
why are these the same? so in both cases, alert or otherwise we will query for Systems/System.Embedded.1, is that the intention, or should we check the hwt